xpdacq/xpdacq.py

```diff
@@ -49,7 +49,13 @@ def _update_dark_dict_list(name, doc):
     dark_dict = {'acq_time': acq_time, 'exposure': light_cnt_time,
                  'timestamp': doc['time'], 'uid': doc['run_start']}
     dark_dict_list.append(dark_dict)
-    glbl['_dark_dict_list'] = dark_dict_list  # update glbl._dark_dict_list
+    if doc['exit_status'] == 'success':
+        print('dark frame complete, update dark dict')
+        dark_dict_list.append(dark_dict)
+        glbl['_dark_dict_list'] = dark_dict_list  # update glbl._dark_dict_list
+    else:
+         print("INFO: dark scan was not successfully executed.\n"
```

Comment:
In general I want our docstrings to be somewhat user-facing/user-helping, especially error messages.  There is nothing more frustrating for a user to find out that something went wrong, with no hint about whether that matters and what to do next.

First, I would say that a failed dark warrants being a WARNING than INFO, doesn't it?  If it failed there must be something wrong.  If anything the print statement from the successful dark should be an INFO:   I also worry about chatter from the software when things are going smoothly....if the software is always outputting statements reassuring the user that things are going well, they won't read them, and then they will miss statements when things didn't go properly.  

Ideally there would be a way that statements went to a log file that is saved somewhere and could be reviewed when desired, but suppressed from being seen by the user under normal circumstances.

Then when there is a warning, please always think....if I were the user, what should I do if I see this warning "dark scan was not successfully executed."?